legacy/src/api/payment-requests/payment-requests.md

```diff
@@ -69,10 +69,10 @@ version (documented on this page) and the "legacy" version (documented at
 | patronCodeId         | String             | The id of a [Patron Code][] the payment request is attached to.                                                                                                          |
 | barcode              | String             | [Scanned Code][] used to create the payment request.                                                                                                                     |
 | expirySeconds        | Number             | The expiry seconds used to configure the payment request expiry.                                                                                                         |
-| lineItems            | Array              | **EXPERIMENTAL** The [Line Items](#line-item) being paid for.                                                                                                            |
```

Comment:
I have no problem with the experimental tag being removed but it would probably be better to do it in a seperate PR next time.

Reply:
That was my bad